src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -57,115 +57,94 @@
  */
 public abstract class AbstractEntityAIGuard<J extends AbstractJobGuard<J>, B extends AbstractBuildingGuards> extends AbstractEntityAIFight<J, B>
 {
+    /**
+     * After this amount of ticks not seeing an entity stop persecution.
+     */
+    protected static final int                          STOP_PERSECUTION_AFTER         = TICKS_SECOND * 10;
+    /**
+     * Interval between guard task updates
+     */
+    protected static final int                          GUARD_TASK_INTERVAL            = 100;
```

Comment:
could you revert the reordering of all this stuff in this class? I'm working on that too atm, and this will give me a lot of merge conflicts

Reply:
This is still untested so it will take time until it will get merged (so yours is probably going to be merged first and then I can resolve the conflicts)

Reply:
This is still untested so it will take time until it will get merged (so yours is probably going to be merged first and then I can resolve the conflicts)